tests/kafkatest/utils/remote_account.py

```diff
@@ -29,4 +31,28 @@ def line_count(node, file):
     if len(out) != 1:
         raise Exception("Expected single line of output from wc -l")
 
-    return int(out[0].strip().split(" ")[0])
\ No newline at end of file
+    return int(out[0].strip().split(" ")[0])
+
+def scp(source_node, source_path, target_node, target_path):
+    """ Copy file from source node to destination node. Uses a temporary file
+    on local node, since there is no mechanism to copy directly between two remote nodes.
+    :param source_node: source node to copy from
+    :param source_path: path on the source node to copy from
+    :param target_node: destination node
+    :param target_path: path on the destination node to copy to
+    """
+    local_temp_dir = "/tmp"
+    try:
+        local_temp_dir = tempfile.mkdtemp(dir=local_temp_dir)
+    except OSError as e:
+        raise Exception("Failed to create temporary local directory to scp %s to %s: %s" % (source_path, target_path, e.strerror))
+
+    local_temp_file = os.path.join(local_temp_dir, "tmpfile")
```

Comment:
Maybe this copy block should be wrapped in `try/except/finally` so you can be sure to clean up temp files even if a copy phase fails. Then probably raise exception or rethrow if there was a problem copying

Also, for the cleanup in the finally block, consider
`shutil.rmtree(path, ignore_errors=True)`